gui/qt/seed_dialog.py

```diff
@@ -131,6 +133,13 @@ def __init__(self, seed=None, title=None, icon=True, msg=None, options=None, is_
             self.seed_warning.setText(seed_warning_msg(seed))
         self.addWidget(self.seed_warning)
 
+    def initialize_completer(self):
+        english_list = Mnemonic('en').wordlist
+        old_list = electrum.old_mnemonic.words
+        wordlist = english_list + list(set(old_list) - set(english_list))
```

Comment:
maybe this is easier to read:
`wordlist = list(set(old_list) | set(english_list))`

Reply:
but if you are doing that to have the words of the new list appear before the words of the old, then disregard

Reply:
It was more to concatenate both lists without having duplicates. Because we're not doing any sort of binary search or tree data structure, the order of the list doesn't matter.

Reply:
What does the order of the suggestions presented to the user depend on? Is that independent of the order in this list here?